CMSgov/onemac
# remove waiver family from package dashboard

Story: https://qmacbis.atlassian.net/browse/OY2-21271
Endpoint: See github-actions bot comment

### Details

remove waiver family column option from package dashboard

### Changes

- removed waiver family column definition and associated render helper method

### Test Plan

1. Login as state submitter
2. Navigate to package dashboard
3. Verify Waiver Family # doesnt appear as a column in the grid or in the show/hide or filter windows
